cli/source_maps.rs

```diff
@@ -134,7 +138,7 @@ pub fn get_orig_position<G: SourceMapGetter>(
             let source_line = if let Some(source_line) = maybe_source_line {
               Some(source_line.to_string())
             } else if let Some(source_line) = getter.get_source_line(
-              original,
+              &file_name,
```

Comment:
The `original` source name written by tsc in the source map object is sometimes reduced to just the base name (#6542). I don't know when exactly this occurs but errors in remote imports are a reliable reproduction.

Passing that deformed specifier to the source map getter makes it fail and fall back to using the transpiled source line which doesn't align with the line and column numbers (#6965). I believe this is also responsible for adding the `<`/`>` characters (#9443).

Reply:
> The unmapped `file_name` comes directly from V8. This actually doesn't need to be mapped, because the module specifiers we give to V8 are already the original user-written ones.

Oh no, this isn't the case for userspace source maps... I think we shouldn't render a source line in this case.

Reply:
Just a heads up!

Came across a little caveat when trying to use this in coverage mapping; token.get_source was populated with an empty string so ended up doing the same as here and going with the known filename instead.

Reply:
@nayeemrmn That would be a great comment to have here in the source code.

Reply:
LGTM otherwise - thank you for the fix!